libs/sdk-ui-vis-commons/src/legend/StaticLegend.tsx

```diff
@@ -33,18 +34,23 @@ export class StaticLegend extends React.PureComponent<IStaticLegendProps> {
     public static defaultProps: Partial<IStaticLegendProps> = {
         buttonOrientation: "upDown",
         paginationHeight: STATIC_PAGING_HEIGHT,
+        onPageChanged: (_page: number) => {},
```

Comment:
Is it necessary to have underscore here?

Reply:
When keeping wihtout underscore, it's complaining about unused param. Removing whole param worked.